recipe/build.sh

```diff
@@ -16,14 +16,18 @@ mkdir ../build
 cd ../build
 cmake $src_dir \
          -DCMAKE_INSTALL_PREFIX=$PREFIX \
+         -DCMAKE_OSX_DEPLOYMENT_TARGET=10.10 \
          -DENABLE_JPG=1 \
          -DENABLE_NETCDF=1 \
          -DENABLE_PNG=1 \
          -DENABLE_PYTHON=0 \
          -DENABLE_FORTRAN=0
 
+if [[ $(uname) == Darwin ]]; then
+  export MACOSX_DEPLOYMENT_TARGET=10.10
```

Comment:
I am can't help you much here as I don't know the first thing about OS X. Hopefully @jakirkham will chime in and help us. However, if this line is needed, it would be nice to put it inside the if-clause above. Or you cannot export that before calling `cmake`?